ipatests/test_integration/test_installation.py

```diff
@@ -479,3 +479,22 @@ def test_reserved_ip_as_forwarder(self):
         exp_str = ("Invalid IP Address 0.0.0.0: cannot use IANA reserved "
                    "IP address 0.0.0.0")
         assert exp_str in cmd.stdout_text
+
+
+class TestIPAServerInstallUnattendedMode(IntegrationTest):
+    """
+    Test ipa-server-install when mandatory params not specified.
+
+    When installing ipa-server in unattended mode (i.e -U option),
+    some manadatory params should be specified with the install
```

Comment:
nitpick but there is a typo (man**a**datory should be manadatory)

Reply:
Okay . I will do that